tests/api2/test_audit_alerts.py

```diff
@@ -0,0 +1,127 @@
+import pytest
+
+from middlewared.test.integration.utils import call, ssh, mock
+from time import sleep
+
+
+def setup_test(alert_class, alert_key, path):
```

Comment:
You can turn this into a contextmanager to handle setup and teardown in the same function